### DISCUSSION TITLE
Filter records by "today"

### DISCUSSION THREAD

**Discussion by rickysullivan** [directus/directus#17368] [Archive]
I'm trying to create a custom view where I show records that have dates as today:

![image](https://user-images.githubusercontent.com/437480/216032819-65255549-dd85-4caa-a2fc-85da28869e7d.png)

The two fields are 'date' fields in the database.

Here is the filter as shown in the DB:

```{"_or":[{"_and":[{"delivery_date":{"_gt":"$NOW(-1 day)"}},{"delivery_date":{"_lt":"$NOW(+1 day)"}}]},{"_and":[{"pickup_date":{"_gt":"$NOW(-1 day)"}},{"pickup_date":{"_lt":"$NOW(+1 day)"}}]}]}```

This is my table when no filters are applied:

![image](https://user-images.githubusercontent.com/437480/216033450-e6695e82-2bd1-4b1b-922f-b5fc5f0813b7.png)

(Today being the 2nd of Feb)

**Comment by br41nslug** [comment]
First thing: today is still the 1st of Feb here (on UTC at least) 😬 so that would make sense to see no results.
You mention you're only dealing with "Date"s not full DateTime's in that case you can check directly against `$NOW` because in a date that would be today:
![image](https://user-images.githubusercontent.com/9389634/216035849-7f0b2cba-d9fd-497d-8236-fd6c7610e015.png)

**Reply by rickysullivan** [reply]
Ahh ok. Yes, I see it's UTC. So, setting a date on my Job is actually setting it to UTC. This is a problem.

There doesn't seem to be a timezone setting for Directus, so I assume the date picker is just picking dates in UTC zone.

I'm using Supabase to host, they recommend:

> Every Supabase database is set to UTC timezone by default. We strongly recommend keeping it this way, even if your users are in a different location. This is because it makes it much easier to calculate differences between timezones if you adopt the mental model that "everything in my database is in UTC time".

https://supabase.com/docs/guides/database/managing-timezones

I wonder how I can allow my admin team to pick a local date, but have it convert to UTC.

**Reply by rijkvanzanten** [reply]
> Ahh ok. Yes, I see it's UTC. So, setting a date on my Job is actually setting it to UTC. This is a problem.
> 
> 
> 
> There doesn't seem to be a timezone setting for Directus, so I assume the date picker is just picking dates in UTC zone.
> 
> 
> 
> I'm using Supabase to host, they recommend:
> 
> 
> 
> > Every Supabase database is set to UTC timezone by default. We strongly recommend keeping it this way, even if your users are in a different location. This is because it makes it much easier to calculate differences between timezones if you adopt the mental model that "everything in my database is in UTC time".
> 
> 
> 
> https://supabase.com/docs/guides/database/managing-timezones
> 
> 
> 
> I wonder how I can allow my admin team to pick a local date, but have it convert to UTC.

That's the difference between a timestamp and a datetime :) time stamps save the timezone info as well, so that should help with this

**Reply by rickysullivan** [reply]
But it then makes the date picker become a date and time picker. I just need a date.
